File: OrderFoodApi/OrderFoodApi/Controllers/Shared/ResponseData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace OrderFoodApi.Controllers.Shared
{
    public class ResponseData
    {
        public int Status { get; set; }
        public object Data { get; set; }
        public string Message { get; set; }

        public ResponseData(int status, object data = null, string message = null)
        {
            Status = status;
            Data = data;
            Message = message;
        }
    }
}

File: OrderApp/app/src/main/java/com/example/trandainhan/orderapp/api/ResponseData.java
package com.example.trandainhan.orderapp.api;


public class ResponseData<T> {
    public int status;
    public T data;
    public String message;

    public ResponseData(int status, T data, String message) {
        this.status = status;
        this.data = data;
        this.message = message;
    }

    public ResponseData(int status, T data) {
        this.status = status;
        this.data = data;
    }
}
